tests/components/smarttub/test_climate.py

```diff
@@ -72,3 +74,7 @@ async def test_thermostat(coordinator, spa, hass, config_entry):
         blocking=True,
     )
     # does nothing
+
+    spa.get_status.side_effect = smarttub.APIError
+    await hass.helpers.entity_component.async_update_entity(entity_id)
```

Comment:
It's best to move time forward to force an update from the coordinator. We can use `async_fire_time_changed` helper.

https://github.com/home-assistant/core/blob/b2df9aaaf1060916e8484260d0914ec3872581e3/tests/common.py#L364-L366